test/boombox_test.exs

```diff
@@ -68,48 +68,60 @@ defmodule BoomboxTest do
          "output.mp4"
        ], "bun_hls_webrtc.mp4", []},
     hls_mpegts_mp4: {[@bbb_hls_mpegts_url, "output.mp4"], "bun_hls_mpegts.mp4", []},
-    mp4_srt_mp4: {[@bbb_mp4, {:srt, "srt://127.0.0.1:9710"}, "output.mp4"], "bun10s.mp4", []},
+    mp4_srt_mp4: {[@bbb_mp4, {:srt, "srt://127.0.0.1"}, "output.mp4"], "bun10s.mp4", []},
     mp4_a_srt_mp4:
-      {[@bbb_mp4_a, {:srt, "srt://127.0.0.1:9710"}, "output.mp4"], "bun10s.mp4",
-       [kinds: [:audio]]},
+      {[@bbb_mp4_a, {:srt, "srt://127.0.0.1"}, "output.mp4"], "bun10s.mp4", [kinds: [:audio]]},
     mp4_v_srt_mp4:
-      {[@bbb_mp4_v, {:srt, "srt://127.0.0.1:9710"}, "output.mp4"], "bun10s.mp4",
-       [kinds: [:video]]},
+      {[@bbb_mp4_v, {:srt, "srt://127.0.0.1"}, "output.mp4"], "bun10s.mp4", [kinds: [:video]]},
     mp4_srt_mp4_with_auth:
       {[
          @bbb_mp4,
          quote(
-           do:
-             {:srt, "srt://127.0.0.1:9710",
-              [stream_id: "some_stream_id", password: "some_password"]}
+           do: {:srt, "srt://127.0.0.1", [stream_id: "some_stream_id", password: "some_password"]}
          ),
          "output.mp4"
        ], "bun10s.mp4", []},
     mp4_a_srt_mp4_with_auth:
       {[
          @bbb_mp4_a,
          quote(
-           do:
-             {:srt, "srt://127.0.0.1:9710",
-              [stream_id: "some_stream_id", password: "some_password"]}
+           do: {:srt, "srt://127.0.0.1", [stream_id: "some_stream_id", password: "some_password"]}
          ),
          "output.mp4"
        ], "bun10s.mp4", [kinds: [:audio]]},
     mp4_v_srt_mp4_with_auth:
       {[
          @bbb_mp4_v,
          quote(
-           do:
-             {:srt, "srt://127.0.0.1:9710",
-              [stream_id: "some_stream_id", password: "some_password"]}
+           do: {:srt, "srt://127.0.0.1", [stream_id: "some_stream_id", password: "some_password"]}
```

Comment:
I am not sure if I like the fact that now the endpoints we use in tests definition are different than Boombox endpoints itself (i.e. in Boombox we have `srt://<host>:<port>` input, and in tests we use `srt://<host>`)